frontend/app/API.js

```diff
@@ -1,63 +1,6 @@
 import { browserHistory } from 'react-router'
 import Raven from 'raven-js'
 
-const parse = {
```

Comment:
I think that could be done with default state in components in a way that's less opaque